### ISSUE TITLE
Proprtionality: clarification

### ISSUE THREAD

**Issue by Hurstharrier** [best-practice-and-impact/aqua_book_revision#107]
From Phillipa R

"Precision of the analysis outputs" I'm not sure what this means, can we be more specific about how this effects the assurance

**Comment by rebecca-wodcke-gad** [comment]
| Precision of the analysis outputs | Imprecise analysis can require different assurance than precise analysis, e.g. because of inherent limitations of the analytical technique, or lack of data on assumptions. |
@Hurstharrier perhaps change it to:

| Level of precision required in outputs" | Imprecise analysis can require different assurance than precise analysis due to accepted uncertainty in the results

What do you think?

**Comment by Hurstharrier** [comment]
@irisoren-sg  have been thinking about this. Most analysis is imprecise. Seems to me that the points here are:
- Most analysis produces imprecise results i.e. there is uncertainty. This may be caused by imprecision in the data, assumptions or the analytical method.
- The questions are: is the analysis is fit for purpose because of the level of uncertainty and what can be done about it. 
- Examine the effect of the uncertainties on the analytical outputs for example using sensitivity analysis.
- Identify ways to reduce the imprecision on the main drivers of the uncertainty in the analysis.

What do you think?

**Comment by rebecca-wodcke-gad** [comment]
@Hurstharrier I agree with these points, but this table is a list of factors that should be considered when determining the appropriate the level of assurance. I'm not sure your bullets fit with this?

I think the initial aim of this point was "if its just a back of the envelope check, that everyone agrees is just rough, then you don't need to do as much assurance as it the calculation is working something out precisely".

If we can't explain it well, I don't think it's essential, there are quite a few other points in the list, so could just remove it.

**Comment by Hurstharrier** [comment]
@rebecca-wodcke-gad  thanks for this. On the lower precision back of the envelope calculations. My experience is that :
- as you say all need to know and agree that you what you are doing is rough
- this may mean using a simplified model (when you could conceivably develop a richer) and simplified assumptions
- this means that the level of precision is lower as a result of using simplifying methods, assumptions etc.
-  in turn this usually makes the task of assuring the work easier as you have a less complicated model
- however I don't think it means not doing the assurance just that it will take less time.

So in a nutshell I think we could say "Lower precision analysis often uses simplified assumption, models and data." In practice the assurance approach is the same but will take less time".

What do you think?
